python/vowpalwabbit/pyvw.py

```diff
@@ -501,6 +501,25 @@ def get_weight(self, index, offset=0):
         """
         return pylibvw.vw.get_weight(self, index, offset)
 
+    def get_weight_from_name(self, feature_name, namespace_name=" "):
```

Comment:
another fn that could be useful would be get_weight_from_interaction and ask for two features/namespaces, we just need to figure out the exact hash formula for interactions

Reply:
not needed for this PR, it's just a thought

Reply:
> another fn that could be useful would be get_weight_from_interaction and ask for two features/namespaces, we just need to figure out the exact hash formula for interactions


Yes that would be quite useful !